### ISSUE TITLE
Skip day view and get to booking entry from month view

### ISSUE THREAD

**Issue by campbell-m** [meeting-room-booking-system/mrbs-code#3629]
(Transferred from the general mailing list)

Is there any hint, trick, hack out there which directs the user to the booking entry from the month view, skipping the day view

Background, rooms are booked on a daily basis. So I have one period per day, and the day view is not really necessary.

Likewise, after saving the booking, the user shall be redirected to the month view. Also, clicking on a booked entry in the month view shall get the user to the booking details, not the day view.

**Comment by campbell-m** [comment]
I think this would be a useful general feature (ie if there is only one slot in the day, go straight to edit_entry) and I will add it to the code.

**Comment by campbell-m** [comment]
I've now added this in [29c429b](https://github.com/meeting-room-booking-system/mrbs-code/commit/29c429b92b4777b31d1b7fc6a306e07e09332242).  I've generalised it with a config setting `$max_slots_direct_to_booking` (default 1).

**Comment by campbell-m** [comment]
I've just realised that clicking on a booking doesn't do what you want.  I'll take a look at that.

**Comment by campbell-m** [comment]
I've now fixed that in [3442d54](https://github.com/meeting-room-booking-system/mrbs-code/commit/3442d54b0a091feb75569b5c725d63f5c7433701).  I've realised that it's not going to be easy to go direct to the booking if there's more than one slot per day, so I've changed the config setting to an enabling setting, `$view_all_always_go_to_day_view` default `false`.  But I think the code should now do what you want it to do.
